tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoDump.java

```diff
@@ -186,6 +186,7 @@ private static void dumpDatabase(TajoClient client, String databaseName, PrintWr
         } else {
           writer.write(DDLBuilder.buildDDLForBaseTable(table));
         }
+
         if (table.hasPartition()) {
```

Comment:
Since we don't support partition dump yet, the whole if block should be removed or commented out. 
In addition, you need to print an warning message for users to know that partition dump and restore are not supported yet.